test/scraped/json_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+require 'pry'
+
+describe Scraped::JSON do
+  describe 'accessing JSON via nokogiri' do
```

Comment:
" via Nokogiri" doesn't seem like it belongs here! Could probably drop this `describe`, since it doesn't add much extra context to the previous one, and just have the `describe Scraped::JSON`, perhaps?